src/windows/windows_sys.rs

```diff
@@ -55,16 +55,18 @@ extern "system" {
 }
 #[link(name = "kernel32")]
 extern "system" {
-    pub fn CreatePipe(
-        hreadpipe: *mut HANDLE,
-        hwritepipe: *mut HANDLE,
-        lppipeattributes: *const SECURITY_ATTRIBUTES,
-        nsize: u32,
-    ) -> BOOL;
+    pub fn OpenSemaphoreA(dwdesiredaccess: u32, binherithandle: BOOL, lpname: PCSTR) -> HANDLE;
```

Comment:
Please update https://github.com/rust-lang/cc-rs/blob/main/dev-tools/gen-windows-sys-binding/windows_sys.list and then run the crate to generate this file, this file is not meant for modification directly.

Reply:
Ah, I was wondering where that file was - thanks!